openshift/ci/testdata/memcached-operator/requirements.yml

```
@@ -0,0 +1,10 @@
+---
+collections:
+  - name: operator_sdk.util
+    version: "0.5.0"
+  - name: kubernetes.core
+    version: "2.4.0"
```

```suggestion
    version: "2.4.2"
```
We need to test the kubernetes.core v2.4.2: https://github.com/openshift/ansible-operator-plugins/pull/6

---

I was thinking about merging this PR first, and then https://github.com/openshift/ansible-operator-plugins/pull/6 so that we are sure that the bump does not break the e2e

---

Are you planning to make the bump in the testdata requirements.yaml file in #6?

---

Yes, that's what I'm thinking

---

Ack. We need this then: https://github.com/openshift/ansible-operator-plugins/pull/12#issuecomment-2288121212

---

Added 
```
COPY --from=builder /ansible/memcached-operator/requirements.yml ${HOME}/requirements.yml
RUN ansible-galaxy collection install -r ${HOME}/requirements.yml \
 && chmod -R ug+rwx ${HOME}/.ansible
```
to catch dependency mismatch